app/build.gradle

```diff
@@ -180,7 +179,11 @@ dependencies {
       'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.1',
       'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.4.1',
       'org.mockito:mockito-core:2.7.22',
-      'com.github.oppia:android-spotlight:ebde38335bfb56349eae57e705b611ead9addb15'
+      'com.github.oppia:android-spotlight:ebde38335bfb56349eae57e705b611ead9addb15',
+
+      //Sambhrant Tiwari
```

Comment:
Please remove the comment. All attributions are handled by git when you make a commit with your username.